myhpi/search/templates/search/search_result.html

```diff
@@ -9,7 +9,7 @@ <h5 class="text-break">{{ result.title }}</h5>
                 {{ result.specific.date|date:"SHORT_DATE_FORMAT" }}
             </time>
```

Comment:
Is it intentionally that the divider in the user menu seems to be gone?

Old menu for comparison:
![grafik](https://github.com/user-attachments/assets/1bcbf92b-3916-4f64-b1c6-5a075fa43ad1)